<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Find:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Users</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QQuickWidget" name="quickView">
         <property name="resizeMode">
          <enum>QQuickWidget::SizeRootObjectToView</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QGroupBox" name="logBox">
      <property name="title">
       <string>Log</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QPlainTextEdit" name="logSection">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="loadPbt">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Download/Update</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>385</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSetup_Ldap">
    <property name="title">
     <string>Servers</string>
    </property>
    <widget class="QMenu" name="menuLdap">
     <property name="title">
      <string>LDAP</string>
     </property>
     <addaction name="actionLdapConnect"/>
    </widget>
    <addaction name="menuLdap"/>
    <addaction name="actionWTS"/>
    <addaction name="separator"/>
    <addaction name="actionUserInput"/>
    <addaction name="actionCurrentServer"/>
   </widget>
   <widget class="QMenu" name="menuRDp">
    <property name="title">
     <string>RDP</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSRDp">
    <property name="title">
     <string>SRDP</string>
    </property>
   </widget>
   <addaction name="menuSetup_Ldap"/>
   <addaction name="menuRDp"/>
   <addaction name="menuSRDp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUserInput">
   <property name="text">
    <string>User Input</string>
   </property>
  </action>
  <action name="actionCurrentServer">
   <property name="text">
    <string>Current Server</string>
   </property>
  </action>
  <action name="actionLdapConnect">
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>LdapConnect</string>
   </property>
  </action>
  <action name="actionWTS">
   <property name="text">
    <string>WTS</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
